### PR TITLE
If site_url is not present, default to mink's base_url

### PR DESCRIPTION
**Pull request by stephenharris**
## Description
When the `site_url` is not present in `behat.yml`, default to `mink.base_url`.

## Motivation and context
If a `site_url` is not provided in the config then accessing `wp-admin/` or `wp-login.php` won't work: https://github.com/paulgibbs/behat-wordpress-extension/blob/master/src/Context/RawWordpressContext.php#L53. A `site_url` is necessary when WordPress is not installed at the `base_url`, but otherwise we can just default to `base_url` so that the URL need not be duplicated in the `behat.yml`.

## How has this been tested?
Manually inspected. It will be covered by the test suite in a forthcoming PR.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by paulgibbs** [comment]
@stephenharris Can you hold off merging this please? I want to test it against my configs after work. Need some time to digest the issue.

**Comment by paulgibbs** [comment]
Also, thanks for completing the new PR template nicely! Feel free to adjust it in the repo if you think it's too wordy or something.

**Comment by paulgibbs** [comment]
OK - at Human Made, we're using this on one of our client projects, and we hadn't run into this despite accessing a wp-admin path, so I wanted to check. We're on a slightly older version so that's why I hadn't seen this bug yet.

Your fix/PR looks super!

**Comment by stephenharris** [comment]
Just to demonstrate the bug, here's the screenshot I got when working on a PR: https://github.com/stephenharris/behat-wordpress-extension/commit/d84c7875af3c570cb6a013f0cfb2d51df972fefd (The history has been rewritten to include this PR):

logged-in-failed
